Tools/autotest/autotest.py

```diff
@@ -200,6 +200,21 @@ def run_step(step):
     if step == "prerequisites":
         return test_prerequisites()
 
+    if step == 'build.All':
```

Comment:
Yeah, I was initially trying to squash all the build.<Vehicle> things together as well, via some substring matching. This needed all the other build targets to be above. It came not to pass however, because of the incosistency of naming going on with rover and heli in particular. So this is a remain from that, is it a big deal?